jcl/src/java.base/share/classes/jdk/internal/misc/Unsafe.java

```diff
@@ -6614,36 +6618,175 @@ private static char convEndian(boolean isBigEndian, char value) {
 	 */
 	public native long getObjectSize(Object o);
 
+	public native boolean isClassFlattened(Class<?> clz);
```

Comment:
The internal helper method should be private.

Reply:
fixed in https://github.com/eclipse-openj9/openj9/commit/c54f1c95bd07867c80bf9bdc9e882849927b4c93